pkgs/applications/science/logic/z3/default.nix

```diff
@@ -1,10 +1,10 @@
 { lib
 , stdenv
+, cmake
 , fetchFromGitHub
 , python
```

Comment:
Since https://github.com/NixOS/nixpkgs/pull/154687 this needs to be python2 or python3.